plugins/record-hook/src/record.utils.ts

```
@@ -5,34 +5,65 @@ import { logError, logInfo } from '@flatfile/util-common'
     }
+  })
+
+  // Remove unchanged values
+  unchangedKeys.forEach((key) => {
+    delete values1[key]
+    delete values2[key]
+  })
+
+  // Clean up empty values object
+  if (Object.keys(values1).length === 0) {
+    delete obj1.values
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)